devito/dimension.py

```diff
@@ -313,39 +382,48 @@ def symbolic_end(self):
         """
         return self.parent.symbolic_end
 
-    def argument_defaults(self, size=None, alias=None):
+    def _arg_defaults(self, start=None, size=None, **kwargs):
         """
         Returns a map of default argument values defined by this symbol.
 
-        :param size: Optional, known size as provided by data-carrying symbols
+        :param start: Optional, known starting point as provided by
+                      data-carrying symbols.
+        :param size: Optional, known size as provided by data-carrying symbols.
```

Comment:
No. In fact, ``t_M`` it's not supported even in current master. I actually wanted to drop this ``t_m/t_M`` thing, to force users use ``time_m/time_M``, as ``t`` is a derived dimension and it makes no sense to specify a min/max point

I haven't dropped it yet because I'd have had to go again through the entire test suite and make changes accordingly

Reply:
Not what I'm talking about. If all `TimeFunction`s in an `Operator` are with `save=False`, the `Operator` does not have a default value for `time_M` or `t_M`.

Reply:
which is ``master``'s current behavior, correct?

Reply:
Do you understand that I am asking for one more line in some docstring somewhere?

Reply:
I think I understand now

Reply:
Added a proper docstring to ``op.apply`` including, among the various things, what you pointed out here.